src/main/java/Event.java

```diff
@@ -0,0 +1,18 @@
+import static com.sun.beans.introspect.PropertyInfo.Name.description;
+
+public class Event extends Task{
+    private String at;
+
+    public Event (String description, String at){
+        super();
+        this.at = at;
+    }
+
+    public String getAt(){
+        return at;
+    }
+
+    public String toString() {
+        return "[E] " + super.toString() + description + " (at: " + at + ")";
+    }
+}
```

Comment:
It is usually a practise to use `this.method()` or `this.variable` when that `method` or `variable` belongs to the Class.

This change can be made at other places as well :)

```suggestion
    public String getAt(){
        return this.at;
    }

    public String toString() {
        return "[E] " + super.toString() + this.description + " (at: " + this.at + ")";
    }
}
```